environment.yaml

```
@@ -0,0 +1,12 @@
+# This file lists the dependencies needed to build and test all of echemdb-app.
+# Create a conda environment with these dependencies preinstalled with:
+# conda env create -f environment.yaml
+name: echemdb_app
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - numpy>=1.26.4,<2.0.0
+  - pip>=24
+  - streamlit>=1.36.0,<2.0.0
+  - unitpackage>=0.8.4,<0.9.0
```

```suggestion
dependencies:
  - numpy>=1.26.4,<2.0.0
  - pip>=24
  - streamlit>=1.36.0,<2.0.0
  - streamlit-extras>=0.4.3,<1.0.0
  - unitpackage>=0.8.4,<0.9.0
```

Hier gibt es ein Problem 😞 Wir benutzen auch die 3rd Party Komponente `streamlit-navigation-bar` ich hab die unter den conda packages nicht gefunden, man müsste also nach dem `conda install` nochmal `streamlit-navigation-bar==3.3.0` ausführen

---

its now included. You can add other relevant things the same way.